.github/workflows/bookkeeping.yml

```diff
@@ -17,20 +17,20 @@ jobs:
       matrix:
         test_type: [unit,
                     api,
-                    lhcPeriods,
```

Comment:
Why this change? I think it follows our js file name convention, even if I prefer your way I think we should keep the original.

Reply:
I thought it resolves issue with hanged jobs: 
![image](https://github.com/user-attachments/assets/1f2e00be-3e1a-446b-9d8a-5a31a5f73311)
, but apparently no

Reply:
but following instruction is `"COMPOSE_PROJECT_NAME=${{ matrix.test_type }}_app"` `test_type` cannot has upercase letters

Reply:
@xsalonx is referring to :
https://docs.docker.com/compose/project-name/#set-a-project-name

Reply:
The `hanging jobs` are due to the fact that GH PR rules are waiting for those jobs names but because you are changing them, they are never being created.
I will update the rules accordingly